docset/winserver2022-ps/configci/New-CIPolicy.md

```diff
@@ -539,6 +541,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -MultiplePolicyFormat
+Indicates that this cmdlet should create a policy in multiple policy format as opposed to a single policy format. 
+Refer to [Create WDAC policies in Multiple Policy Format](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/deploy-multiple-windows-defender-application-control-policies#creating-wdac-policies-in-multiple-policy-format) for the difference between the policy formats. 
```

Comment:
Can you remove the `en-us` from the URL?

Reply:
Removed in latest commit (https://github.com/MicrosoftDocs/windows-powershell-docs/pull/2483/commits/05ecd0c211df5df115e5821224eb3b87322e6a43)